system-test/logging-winston.ts

```diff
@@ -28,6 +31,17 @@ const logging = new Logging();
 const LoggingWinston = require('../src/index').LoggingWinston;
 const LOG_NAME = 'winston_log_system_tests';
 
+const UUID = uuid.v4();
+function message(text: string) {
```

Comment:
I think it would be better to modify the `LOG_NAME` above to include the `uuid` rather than modifying individual messages.

Reply:
Updated.

Reply:
Maybe it is github, but I dont' see the update yet.

Reply:
Ok you should be able to see the updates now.